operator/gitops/argocd/pipeline-service/openshift-pipelines/appstudio-pipelines-runner-clusterrole.yaml

```diff
@@ -0,0 +1,54 @@
+---
+# Any change to this role must be communicated to the devsandbox team
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: appstudio-pipelines-runner-clusterrole
+rules:
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - pipelineruns
+    verbs:
+      - get
+      - list
+      - update
+      - patch
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - taskruns
+    verbs:
+      - get
+      - patch
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - taskruns/status
+    verbs:
+      - patch
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - get
+      - create
+      - patch
+      - delete
+  - apiGroups:
+      - sharedresource.openshift.io
```

Comment:
maybe add a comment that this can go away when the last vestiges of shared resources are removed

Last I understood, https://issues.redhat.com/browse/STONEBLD-867 was the tracker for that, but @Michkov would be the definitive source of truth for that.